lib/bot.js

```diff
@@ -40,6 +42,9 @@ function Bot(options) {
 Bot.prototype.attachListeners = function() {
   this.client.addListener('registered', function(message) {
     logger.debug('Registered event: ', message);
+    for (var i = 0, len = this.autosendcmd.length; i < len; i++) {
```

Comment:
Please use a `.forEach`-loop instead :). If you use `.bind` on the listener callback you can use `this.autoSendCommands` even with the changes I mentioned in the comment above.